bin/cylc-run

```diff
@@ -95,7 +96,11 @@ warm start unless you use special cold-start tasks. See "Suite Start-up" and
             start_point_string = self.args[1]
             if start_point_string == "now":
                 start_point_string = (
-                    datetime.datetime.utcnow().strftime("%Y%m%dT%H%MZ")
+                    datetime.datetime.now().strftime("%Y%m%dT%H%M")
+                )
+            elif start_point_string == "UTCnow":
+                start_point_string = (
+                    datetime.datetime.utcnow().strftime("%Y%m%dT%H%M")
```

Comment:
By the way, the original Z, with the CLI "now" start point was presumably an error? (to assume UTC)